src/Microsoft.Health.Fhir.CosmosDb.UnitTests/Features/Storage/Versioning/CollectionUpgradeManagerTests.cs

```diff
@@ -62,7 +63,7 @@ public CollectionUpgradeManagerTests()
             collectionVersionWrappers.ReadNextAsync(Arg.Any<CancellationToken>())
                 .Returns(Substitute.ForPartsOf<FeedResponse<CollectionVersion>>());
 
-            var updaters = new ICollectionUpdater[] { new FhirCollectionSettingsUpdater(_cosmosDataStoreConfiguration, optionsMonitor, NullLogger<FhirCollectionSettingsUpdater>.Instance), };
+            var updaters = new ICollectionUpdater[] { /*new FhirCollectionSettingsUpdater(_cosmosDataStoreConfiguration, optionsMonitor, NullLogger<FhirCollectionSettingsUpdater>.Instance),*/ };
```

Comment:
We should use the DataPlaneCollectionSetup as it now how the logic from FhirCollectionSettingsUpdater.

Reply:
done